packages/slate/src/models/node.js

```diff
@@ -250,14 +253,13 @@ class Node {
    */
 
   assertNode(key) {
-    const node = this.getNode(key)
+    key = assertKey(key)
 
-    if (!node) {
-      key = assertKey(key)
+    if (!this.hasNode(key)) {
```

Comment:
Because `hasNode` is faster than `getNode`.  `hasNode` is only for whether path exists, while `getNode` additionally contains visit node by path.

Therefore we use `hasNode` is better here.

Reply:
Another reason is `hasNode` is more verbally alike with `assertNode`